CNAME

```diff
@@ -0,0 +1 @@
+opengitops.dev
```

Comment:
Per https://github.com/peaceiris/actions-gh-pages#%EF%B8%8F-add-cname-file-cname I believe the existing file we have here is correct: https://github.com/open-gitops/website/blob/main/public/CNAME. Is this necessary So let's remove this duplicate file from the top level of your PR branch.

Looks like we could also add 
```yaml
with:
    cname: opengitops.dev
```
to the GitHub Pages action workflow instead. Either way should be fine

Reply:
Ah didn't catch that CNAME in public. Will remove my change - prefer to use that than bake it into the workflow, for flexibility.